app/models/account/syncable.rb

```diff
@@ -2,7 +2,7 @@ module Account::Syncable
     extend ActiveSupport::Concern
 
     def sync_later
-        AccountSyncJob.perform_later self
+        AccountSyncJob.perform_later(id)
```

Comment:
Was this a leftover commit?  Or is this still required?

Reply:
Now we have removed `after_commit :sync_account` from Transaction and Valuation so it's not necessary, I reverted the commit!

Reply:
Awesome, thanks!